example-app/src/output.js

```diff
@@ -11,12 +11,18 @@ class Output {
   ];
 
   async serve (req, res) {
-    req.res.locals.searchRequest = {
+    req.res.locals.searchRequestBody = {
       filter: {
         group: ['6e23b09ac3944b3fb1f98b34a9fc33c4','358380575024451b8a0e496d871ad731','455c193b5e044d7c862080fd1087c656']
+      },
+     options: {
+        fields: 'name,description,tags,created,modified,owner,orgContactEmail,extent,license'
       }
     };
 
+    req.res.locals.arcgisPortal = 'https://www.arcgis.com';
```

Comment:
is arcgisPortal suppose to go on req.app.locals?

Reply:
Yes, it's for example app which might not require `arcgisPortal` but good to have it defined here.